code/modules/research/xenobiology/crossbreeding/_misc.dm

```diff
@@ -0,0 +1,170 @@
+	opacity = TRUE
+	timeleft = 100
+
+//Rainbow barrier - Chilling Rainbow
+/obj/effect/forcefield/slimewall/rainbow
+	name = "rainbow barrier"
+	desc = "Despite others' urgings, you probably shouldn't taste this."
+	icon_state = "rainbowbarrier"
+
+//Ration pack - Chilling Silver
+/obj/item/reagent_containers/food/snacks/rationpack
+	name = "ration pack"
+	desc = "A square bar that sadly <i>looks</i> like chocolate, packaged in a nondescript grey wrapper. Has saved soldiers' lives before - usually by stopping bullets."
+	icon_state = "rationpack"
+	bitesize = 3
+	eatverb = "choke down"
+	junkiness = 15
+	filling_color = "#964B00"
+	tastes = list("cardboard" = 3, "sadness" = 3)
+	foodtype = null //Don't ask what went into them. You're better off not knowing.
+	list_reagents = list("stabilizednutriment" = 10, "nutriment" = 2) //Won't make you fat. Will make you question your sanity.
+
+/obj/item/reagent_containers/food/snacks/rationpack/checkLiked(fraction, mob/M)	//Nobody likes rationpacks. Nobody.
+	if(last_check_time + 50 < world.time)
+		if(ishuman(M))
+			var/mob/living/carbon/human/H = M
+			if(H.mind && !H.has_trait(TRAIT_AGEUSIA))
+				to_chat(H,"<span class='notice'>That didn't taste very good...</span>") //No disgust, though. It's just not good tasting.
+				GET_COMPONENT_FROM(mood, /datum/component/mood, H)
+				if(mood)
+					mood.add_event(null,"gross_food", /datum/mood_event/gross_food)
+				last_check_time = world.time
+				return
+	..()
+
+//Ice stasis block - Chilling Dark Blue
+/obj/structure/ice_stasis
+	name = "ice block"
+	desc = "A massive block of ice. You can see something vaguely humanoid inside."
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "frozen"
+	density = TRUE
+	max_integrity = 100
+	armor = list("melee" = 30, "bullet" = 50, "laser" = -50, "energy" = -50, "bomb" = 0, "bio" = 100, "rad" = 100, "fire" = -80, "acid" = 30)
+
+/obj/structure/ice_stasis/Initialize()
+	. = ..()
+	playsound(src, 'sound/magic/ethereal_exit.ogg', 50, 1)
+
+/obj/structure/ice_stasis/Destroy()
+	for(var/atom/movable/M in contents)
+		M.forceMove(loc)
+	playsound(src, 'sound/effects/glassbr3.ogg', 50, 1)
+	return ..()
+
+//Gold capture device - Chilling Gold
+/obj/item/capturedevice
+	name = "gold capture device"
+	desc = "Bluespace technology packed into a roughly egg-shaped device, used to store nonhuman creatures. Can't catch them all, though - it only fits one."
+	w_class = WEIGHT_CLASS_SMALL
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "capturedevice"
+
+/obj/item/capturedevice/attack(mob/living/M, mob/user)
+	if(contents.len)
+		to_chat(user, "<span class='warning'>The device already has something inside.</span>")
+	if(!isanimal(M))
+		to_chat(user, "<span class='warning'>The capture device only works on simple creatures.</span>")
+		return
+	if(M.mind)
+		to_chat(user, "<span class='notice'>You offer the device to [M].</span>")
+		if(alert(M, "Would you like to enter [user]'s capture device?", "Yes", "No") == "Yes")
```

Comment:
As far as I'm aware, the third parameter of the alert proc is the window title. What sort of results did you get when testing?

Reply:
`if(alert(src,"Confirm location.","Template Confirm","Yes","No") == "Yes")
     if(template.load(T, centered = TRUE))`

This is from the map template system. I'm pretty sure the order is correct. Will check once I'm home from work if I'm wrong.